app/frontend/src/features/messages/surfing/HostRequestListItem.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import { Meta, Story } from "@storybook/react";
+
+import { hostRequest } from "../../../stories/__mocks__/service";
+import HostRequestListItem from "./HostRequestListItem";
+
+export default {
+  title: "Messages/HostRequestListItem",
+  component: HostRequestListItem,
+} as Meta;
+
+const Template: Story<any> = (args) => (
+  <>
+    <HostRequestListItem {...args} />
```

Comment:
The loading state of this looks a bit odd, it's showing partially loaded data before the avatar/user is being loaded etc. Wonder if that's just the mocks being weird again...

Reply:
It's because the service mock in storybook adds a 1 second loading time to everything. It could be noticeable in real usage, so I cleaned the skeletons up a bit.